source/scripting_v3/GTA/Game.cs

```diff
@@ -543,5 +543,16 @@ public static int GetProfileSetting(int index)
 		{
 			return Function.Call<int>(Hash.GET_PROFILE_SETTING, index);
 		}
+
+		/// <summary>
+		/// Searches the address space of the current process for a memory pattern.
+		/// </summary>
+		/// <param name="pattern">The pattern.</param>
+		/// <param name="mask">The pattern mask.</param>
+		/// <returns>The address of a region matching the pattern or <see langword="null" /> if none was found.</returns>
+		public static unsafe byte* FindPattern(string pattern, string mask)
```

Comment:
I would expose the method as IntPtr FindPattern(string, string) instead of byte* FindPattern(string, string) if I was the owner of this repo. It’s up to @crosire, though.
VB projects cannot directly use the method if it returns a pointer type. I don’t recommend VB for scripts IMO, but it is true the return type `Entity.MemoryAddress` was changed from `int*` to `IntPtr` in v3 API.

Reply:
Maybe the method should accept the format like this (just like IDA and Cheat Engine support): `B8 ? ? ? ? C3`